package-lock.json

```diff
@@ -2708,37 +2708,6 @@
       "dev": true,
       "license": "BSD-3-Clause"
     },
-    "node_modules/@intlify/bundle-utils": {
```

Comment:
As there is no change in package.json file, so we should not update package-lock file as well.